verifier/src/report_body.rs

```diff
@@ -24,6 +27,17 @@ pub trait Accessor<T>: Debug {
     fn get(&self) -> T;
 }
 
+/// [`Accessor`] for returning Self, i.e. T -> T
```

Comment:
What's the point of this macro? It's only used in one place right now, will it be used more in other places? What's the difference compared to just calling `.clone()`?

Reply:
The main reason for:
```rust
impl Accessor<T> for T {
    fn get(&self) -> T {
        self.clone()
     }
}
```
(and now that I think of it, I realize I can implement a generic impl instead of a macro... will do on a subsequent PR)

Anyway,
In order to be able to pass an instance of `Thing` to the `ThingVerifier` there needs to be an `Accessor` trait for `Thing`.
```
impl<T: Accessor<Thing>> Verifier<T> for ThingVerifier {

    fn verify(&self, evidence: &T) -> ...
```